flink-runtime/src/test/java/org/apache/flink/runtime/state/TaskExecutorLocalStateStoresManagerTest.java

```diff
@@ -53,25 +53,26 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 /** Test for {@link TaskExecutorLocalStateStoresManager}. */
-public class TaskExecutorLocalStateStoresManagerTest extends TestLogger {
+class TaskExecutorLocalStateStoresManagerTest {
```

Comment:
In this class there are tests with public modifiers which could be hardened

Reply:
I somehow missing this. 😞